examples/amp-consent/cmp-vendors.amp.html

```diff
@@ -185,6 +211,30 @@ <h3>Image that is NOT blocked by consent</h3>
         </amp-consent>
         <!-- End didomi example -->
 
+      <!-- iubenda example -->
+      <amp-consent id="iubenda" layout="nodisplay" type="iubenda">
+        <!--
+          If you want to request consent only to EU users then replace "consentRequired": true with "promptIfUnknownForGeoGroup": "eu" -> allows to ask consent only to EU users.
+        -->
+        <script type="application/json">
+          {
+            "consentRequired": true,
+            "promptUI": "iubenda-consent-ui"
```

Comment:
No promptUI is needed. promptUiSrc (configured in cmps.js) must be defined instead. 

It looks like it should be defined with: https://cdn.iubenda.com/cs/test/cs-for-amp.html.

This mean you can also remove `<div id="iubenda-consent-ui" class="popupOverlay">` unless you want this to be your post prompt ui.

Reply:
We need our custom promptUI as we are using custom CSS, as I clarified in the PR description, each user must host his own promptUISrc therefore this is necessary.

Reply:
Additionally, `promptUi` overrides `promptUiSrc`, so you should replace it in this example with `prompUiSrc: https://cdn.iubenda.com/cs/test/cs-for-amp.html`

Reply:
Done.